### PR TITLE
validate organisation selection against valid options

### PR DESCRIPTION
**Pull request by levgorbunov1**
- Validates user selection for organisations against valid options set in configuration.

### CODE REVIEW

**Review by connormaglynn — DISMISSED** [review]
Also worth doing the same validation in the `github_service` class as an extra measure to ensure that the value are valid before actually using them to send the invitation 📬 

Happy for that to be a separate PR though 😁

File: join_github_app/main/services/github_service.py [hunk 1]
```diff
@@ -20,8 +20,11 @@ def __init__(self, org_token: str) -> None:
...
+        valid_orgs = [ organisation.name for organisation in app_config.github.organisations if organisation.enabled ]
         for organisation in organisations:
-            if app_config.github.send_email_invites_is_enabled:
+            if organisation in valid_orgs and app_config.github.send_email_invites_is_enabled:
                 self.github_client_core_api.get_organization(organisation.lower()).invite_user(email=email)
-            else:
+            elif not app_config.github.send_email_invites_is_enabled:
                 logger.info("Not sending invitation for organisation [ %s ] as SEND_EMAIL_INVITES is [ %s ]", organisation, app_config.github.send_email_invites_is_enabled)
+            else:
+                logger.info("Not sending invitation for organisation [ %s ] as selected organisation is invalid", organisation)
```
Comment (tamsinforbes) [inline join_github_app/main/services/github_service.py:30]:
```suggestion
                logger.info("Invitation for organisation [ %s ] not sent as selected organisation is invalid", organisation)
```
I've changed the wording here in my PR; you might need to update with main to pull the other change in

**Review by tamsinforbes — APPROVED** [review]
🌝🌖🚀LGTM🚀🌘🌚

Minor comment to update the logging wording to match the changes in my commit.

**Review by tamsinforbes — APPROVED** [review]
🌝🌖🚀LGTM🚀🌘🌚

**Pull request merged**